translations/monero-GUI-guide_es.md

```diff
@@ -0,0 +1,331 @@
+**(12) Cambiar ubicación:** Usa la interfaz gráfica para introducir una ruta no predeterminada en el sistema a la cadena de bloques.    
+**(13) Mostrar avanzado:** Marca esta casilla para mostrar las opciones avanzadas (indicadores de arranque o inicio de sesión en el nodo).    
+**(14) Indicadores de arranque / Inicio de sesión en el nodo:** Al usar un nodo local, será aquí donde puedas introducir opciones de línea de comando adicionales. Cuando use un nodo remoto, podrás facilitar un nombre de usuario y contraseña en caso de que se requiera autenticación.    
+**(15) Decoraciones personalizadas:** Marca esta casilla para mostrar las decoraciones personalizadas de Monero.    
+**(16) Nivel de registro:** Cambia la verbosidad de los registros de depuración.    
+**(17) Categorías de registro:** Agrega categorías específicas a los registros de depuración.    
+**(18) Altura de creación de cartera:** Cambia la altura de bloque a la que volverá a escanear la billetera. Haz clic en `Guardar` para que los cambios surtan efecto.
+
+## 6.1. Semilla y claves
+
+La pestaña `Semillas y claves` muestra la información mnemotécnicas de su billetera, así como su clave de vista secreta, clave de vista pública, clave de gasto secreta y clave de gasto público.
+
+![seed-keys](/media/black_seed-keys.png)
+
+A continuación, se describen brevemente algunas de las opciones presentes en esta pestaña:
+
+**(1) Semilla mnemotécnica:** [**NO comparta su semilla mnemotécnica con nadie*. Guarde una copia de forma segura.] La semilla mnemotécnica es una frase de 25 palabras que contiene toda la información necesaria para ver y gastar fondos. _Obtén más información sobre las semillas mnemotécnicas en Monero en [este enlace [EN]](https://getmonero.org/resources/moneropedia/mnemonicseed.html)._    
+**(2) Clave de vista secreta:** Las claves de vista secretas permiten al titular ver las transacciones entrantes de su monedero, pero no las salientes. A veces esta funcionalidad es útil para permitir que un tercero pueda auditar y comprobar las transacciones entrantes.    
+**(3) Clave de vista pública:** Las claves de vista públicas se usan para la creación de direcciones ocultas. _Obtén más información sobre las claves en [este enlace [EN]](https://getmonero.org/resources/moneropedia/viewkey.html)._    
+**(4) Clave secreta de gastos:** [**NO comparta su clave secreta de gastos con nadie.**] La clave de gasto secreta se usa para firmar transacciones y debe protegerse con un nivel de seguridad equivalente al de su semilla mnemotécnica.    
+**(5) Clave de gasto público:** La red utiliza la clave de gasto pública para verificar la firma de la imagen de clave que se genera cuando realiza una transacción. Esto evita la presencia del doble gasto ya que la red obliga a que una clave de imagen solo pueda gastarse una vez. _Obtén más información sobre las claves de gasto en [este enlace](https://getmonero.org/resources/moneropedia/spendkey.html)._    
```

Comment:
Just to give it some thoughts, not sure about this translation:

"Clave de gasto pública" or "Clave pública de gasto" 

And the same with the link: https://getmonero.org/es/resources/moneropedia/spendkey.html